### DISCUSSION TITLE
what features do we get with the localhost version

### DISCUSSION THREAD

**Discussion by snozwoz** [mckaywrigley/chatbot-ui#1687] [Help]
Hiya, this looks awesome, and I want to give this a try. 

When I look at your hosted demo version you offer Free (standard chat features) and Pro (reusable chat settings, files uploads etc). 

If I manage to set up this localhost version from your git repro, will this replicate which version feature set, the free or the Pro version?

Thanks in advance, looking forward to giving this a go.

**Comment by briangladu** [comment]
Appears to be the pro version! All paid features appear to be enabled which is amazing and very generous. I went ahead and paid for the Pro online version in support of this amazing project.
